src/run.ts

```diff
@@ -79,5 +79,33 @@ async function runSetup(): Promise<null | Error> {
   core.info(`Successfully setup buf version ${version}`);
   core.info(cp.execSync(`${binaryPath} --version`).toString());
 
+  const bufToken = core.getInput("buf_token");
```

Comment:
After reading [this](https://github.com/bufbuild/buf-setup-action#buf-token) in the readme, and further [here](https://docs.buf.build/bsr/authentication#buf_token) in our docs... the key `buf_token` could get a bit confusing. 
```suggestion
  const bufAPIToken = core.getInput("buf_api_token");
```